gitbot.rb

```diff
@@ -30,6 +30,29 @@ def pr_test(upstream, pr_sha_com, repo, pr_branch)
   git.del_pr_branch(upstream, pr_branch)
 end
 
+def check_if_changes_files_changed(repo, pr)
+  if @changelog_test
+    @j_status = 'success'
```

Comment:
we are adding success without executing any test is this intentional?

Reply:
```
    @j_status = 'success'

    if @pr_files.any? == false
      @j_status = 'failure'
```
Yes, this is by intention: default success, if pr_files is empty set to failure.
I could do an `else` if you want.